docs/content/en/docs/clustermgmt/verify-cluster-image.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Verify Cluster Images"
+linkTitle: "Delete cluster"
+weight: 90
+aliases:
+    /docs/tasks/cluster/verify-cluster-images/
```

Comment:
You don't need the aliases line and the one that follows. In other files, we used them when files were moved from other locations so someone going to the original page would be redirected.